iothub/service/tests/ClientWebSocketTransportTests.cs

```diff
@@ -79,6 +79,7 @@ public async Task ClientWebSocketTransportReadWithoutConnectTest()
 
         // The following tests can only be run in Administrator mode
         [TestMethod]
+        [Timeout(10000)] // if it is going to fail, fail fast. Otherwise it can go on for 4+ minutes. :(
```

Comment:
nit: save this timeout value as a constant.

Reply:
Why?

Reply:
So that we don't end up having to maintain different values everywhere

Reply:
General rule of thumb is to make something a constant if it is referenced from more than one location. In this case, we aren't. The value only matters here. If I found a second method that needed the same value, then I'd consider it.

Reply:
I'm going to resolve this as by design, for now. Hit me up on teams if you want to talk about it. :)